### PR TITLE
reference/configuration: add description about TiDB txn-total-size-limit

### PR DESCRIPTION
**Pull request by tiancaiamao**
See https://github.com/pingcap/docs/pull/2002

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @jackysp @TomShawn

File: reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -243,6 +243,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：0
 + 这个配置只有在 prepared-plan-cache.enabled 为 true 的情况才会生效。在 LRU 的 size 大于 prepared-plan-cache.capacity 的情况下，也会剔除 LRU 中的元素。
 
+### `txn-total-size-limit`
+
++ TiDB 事务大小限制
++ 默认值：104857600
```
Comment (TomShawn) [inline reference/configuration/tidb-server/configuration-file.md]: 是否需要加一个单位？

File: reference/configuration/tidb-server/configuration-file.md [hunk 2]
```diff
@@ -243,6 +243,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ 单个事务中，所有的 key-value 记录的总大小不能超过该限制。注意，如果 `binlog` 开启了，该配置项的值不能超过 104857600(100M)，因为 `binlog` 组件不支持同步的事务过大；如果 `binlog` 没开启，该配置项的最大值不超过 10737418240(10G)。
```
Comment (TomShawn) [inline reference/configuration/tidb-server/configuration-file.md]: `binlog` 是 TiDB Binlog 吗？
```suggestion
+ 单个事务中，所有 key-value 记录的总大小不能超过该限制。注意，如果开启了 `binlog`，该配置项的值不能超过 `104857600` (表示 100MB)，因为 `binlog` 组件不支持同步的事务过大。如果 `binlog` 没开启，该配置项的最大值不超过 `10737418240` (表示 10GB)。
```

Comment (jackysp) [inline reference/configuration/tidb-server/configuration-file.md]: This comment may not be related to the documentation. Should we set a lower limit, if the size is too low, TiDB will not start.

**Comment by TomShawn** [comment]
@tiancaiamao Please address the above comments, thanks!

**Comment by tiancaiamao** [comment]
PTAL @TomShawn @jackysp

File: reference/transactions/overview.md [hunk 3]
```diff
@@ -209,3 +209,5 @@ COMMIT;
 为了使性能达到最优，建议每 100～500 行写入一个事务。
 
 TiDB 设置了单个事务的键值对的总大小不超过 100 MB，这个默认值可以通过配置文件中的配置项 `txn-total-size-limit` 进行修改，最大支持到 10GB。实际的单个事务大小限制还取决于用户的内存，执行大事务时 TiDB 进程的内存消耗大约是事务大小 6 倍以上。
+
+在 4.0 以前的版本，TiDB 限制了单个事务的键值对的总数量不超过 30 万条，在 4.0 之后的版本没有这项限制。
```
Comment (TomShawn) [inline reference/transactions/overview.md:213]: Please update this part in [#2002](https://github.com/pingcap/docs/pull/2002).

Comment (tiancaiamao) [reply reference/transactions/overview.md:213]: There is nothing about it here https://github.com/pingcap/docs/blob/3a61188f3c8cd78369d50f2553edcbbd3273b022/reference/transactions/overview.md

Comment (tiancaiamao) [reply reference/transactions/overview.md:213]: The whole block is missing

Comment (TomShawn) [reply reference/transactions/overview.md:213]: This is not the latest document. This is : https://github.com/pingcap/docs/blob/master/reference/transactions/overview.md.

Comment (TomShawn) [reply reference/transactions/overview.md:213]: @tiancaiamao The missing part is aligned in https://github.com/pingcap/docs/pull/2280. PTAL

**Pull request merged**